### ISSUE TITLE
[elasticsearch] Version support issues

### ISSUE THREAD

**Issue by love01211** [Graylog2/graylog2-server#9706]
## Expected Behavior
Normal use

## Current Behavior
{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"request [/_cat/indices/graylog_*] contains unrecognized parameter: [expand_wildcards]"}],"type":"illegal_argument_exception","reason":"request [/_cat/indices/graylog_*] contains unrecognized parameter: [expand_wildcards]"},"status":400}

## Possible Solution
Upgrade elasticsearch to 7.9，Normal use

## Steps to Reproduce (for bugs)
1.start mongodb
2.start elasticsearch
3.start graylog-server
4.ERROR log

## Your Environment
virtual machine

* Graylog Version: 4.0.1-1
* Java Version:11.0.9
* Elasticsearch Version:7.6.1
* MongoDB Version:4.2.11
* Operating System:CentOS Linux release 7.5.1804 (Core)
* Browser version:Google Chrome 86

**Comment by dennisoelkers** [comment]
Hey @love01211, thanks for reporting this!

I am not sure I understood the issue right. Did upgrading ES to 7.9 fix the issue for you?

**Comment by love01211** [comment]
@jalogisch yes,upgrading ES to 7.9 fix the issue,But I want to know the list of supported versions,官方文档里记录为7

**Comment by Ricky-Hao** [comment]
We encountered this problem too.
Our ElasticSearch version is *7.6.2*, which does not suppurt `expand_wildcards` params on `_cat/indices` API.
```json
{
  "error": {
    "root_cause": [
      {
        "type": "illegal_argument_exception",
        "reason": "request [/_cat/indices/graylog_default_*] contains unrecognized parameter: [expand_wildcards]"
      }
    ],
    "type": "illegal_argument_exception",
    "reason": "request [/_cat/indices/graylog_default_*] contains unrecognized parameter: [expand_wildcards]"
  },
  "status": 400
}
```
Maybe, we should remove this line in code? Or check ES version before set this params.
https://github.com/Graylog2/graylog2-server/blob/f35df42e165ac570b8b27de3f8eeac85e74ed610/graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/cat/CatApi.java#L82

**Comment by Ricky-Hao** [comment]
In official document, `_cat/indices` add `expand_wildcards` param above version `7.6`.
